docs/manuals/core/entities/pipeline-mgt.md

```diff
@@ -87,4 +87,27 @@ existing pipelines.
 A pipeline can be deleted by using `taipy.delete()^` which takes the pipeline id as a parameter. The deletion is
 also propagated to the nested tasks, data nodes, and jobs if they are not shared with any other pipeline.
 
+# Get parent scenarios
+
+To get the parent scenario of a pipeline, you can use the `Pipeline.get_parents()^` method. The method will return a dictionary with the key being the level of the parent ("scenarios") and the value is a set of the parent entities corresponding to the level. You can also use `taipy.get_parents()^`, which takes a child entity such as the task as input, to achieve the same result.
```

Comment:
"parent scenario", but "get_parents()"? Inconsistency here.
And don't bother repeating the method doc.